.env.example

```diff
@@ -0,0 +1,4 @@
+GATSBY_ALGOLIA_APP_ID=
```

Comment:
Q: we already have a few Algolia search boxes implemented - how did we pass these variables before?

Reply:
Yes, we have them implemented and the required variables are directly set to the required places. But, in this specific case, we have the ADMIN_KEY variable which should not be exposed with code. So, it's passed into this variable.

Reply:
Got it, I see that workflow is very different.